Reference/Notifications/SendingAllowedChildren-Notification/index.md

```diff
@@ -0,0 +1,5 @@
+---
+versionFrom: 9.0.0
+versionTo: 10.0.0
+---
```

Comment:
Is there a reason why this was added? 🙈 These versions to and from where only used on our old legacy documentation so it is not needed at all on these current documentations. This file is also part of our old documentation repository 😓